### ISSUE TITLE
Incorrect country recognition for the Airport WiFi card

### ISSUE THREAD

**Issue by OKonnel** [dortania/OpenCore-Legacy-Patcher#328]
The creation of the EFI Build does not correctly recognize the country when setting the patch of the Airport-Bluetooth card. (see attached clip # 01).
My country is Italy, but it is detected and selected as the US
Although Monterey and Big Sur work well, the incorrect indication in the EFI OC (US instead of IT) does not allow the correct selection of the WiFi Channels, that differ between US and Europe (see attached clips # 02 and # 03).
As you can see from the clips, if I manually fix the "config.plist" file the WiFi module is recognised as IT by "System Information" and all the European WiFi channels appear, even if the Local entry incorrectly reports "Unknown" instead CE or EU, etc. With the US country, instead, the "Local" indicate correctly "FCC" (Federal Communications Commission).

![01 Country-detect-error](https://user-images.githubusercontent.com/81856309/123082624-84155f80-d41f-11eb-8dfd-c42c8035f7ca.jpg)

![02 Monterey WiFi US](https://user-images.githubusercontent.com/81856309/123082648-8c6d9a80-d41f-11eb-941d-023a4f4dde03.jpg)

![03 Monterey WiFi IT](https://user-images.githubusercontent.com/81856309/123082671-91324e80-d41f-11eb-9678-bc8d1a16d404.jpg)

**Comment by Ausdauersportler** [comment]
Have you tried the latest nightly [build](https://nightly.link/dortania/OpenCore-Legacy-Patcher/workflows/build-app/main/OpenCore-Patcher.app.zip)?
Just read on Discord that is should have been fixed. Not sure if it made it into the latest upload, though.

**Comment by khronokernel** [comment]
Update OCLP from a native OS without OpenCore booted, will pull correct country code on Broadcom cards

**Comment by OKonnel** [comment]
Thanks, @khronokernel. I don't have a clean "native" System because I only use Big Sur and Monterey on my office iMac, no problem, thanks to your work and the work of your friends.
Therefore, after your suggestion, I proceeded like this:
- I disabled all various Kext in EFI/EFI/OC/config.plist except "SMC-Spoof.kext".
- I restarted the 27'' iMac14,2 late 2013.
- I used OCLP to recreate EFI Build.
- So now OCLP has correctly recognized Italy country (IT), instead US, for the Broadcom module and activated all correct WiFi channels for Italy, even though "Local:", in System Information, still indicates "Unknown" [perhaps due to a bad management of "AirportBrcmFixup.kext"?...].
As I'm not a Python expert, I use the OpenCore Configurator or BBEdit apps to edit "config.plist".
With this indication, I hope to be useful to development and to other nerd / experienced users in the same situation as me.
I always bless you for everything you do and I send you thoughts of happiness and health :)
